Build/Build.cs

```diff
@@ -145,32 +145,37 @@ class Build : NukeBuild
                     cc => cc.SetProjectFile(Solution.Specs.Approval_Tests)));
         });
 
-    Target UnitTests => _ => _
+    Project[] Projects => new[]
+    {
+        Solution.Specs.FluentAssertions_Specs,
+        Solution.Specs.FluentAssertions_Equivalency_Specs
+    };
+
+    Target UnitTestsNetFramework => _ => _
```

Comment:
🤔 Why do you need this separation?

Reply:
For the non-windows jobs it becomes explicit that the windows tests are skipped.

![image](https://user-images.githubusercontent.com/919634/212463796-d7eb86de-bd37-4dd9-b998-37b78ee13b10.png)